docs/provisioners/kustomize.md

```diff
@@ -0,0 +1,355 @@
+# kustomize Provisioner
+
+## Summary
+
+The `kustomize` provisioner is one of the [provisioners](https://github.com/operator-framework/rukpak/tree/main/internal/provisioner) of RukPak.
+It instantiates a given `kustomize+v0` bundle onto a cluster and then install the generated resources
+on the cluster.  The `kustomize+v0` bundle is a [Kustomization](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/).
+It does so by reconciling `Bundle` and `BundleDeployment` types that have
+the `spec.provisionerClassName` field set to `core-rukpak-io-kustomize`. This field must be set to the correct provisioner
+name in order for the `kustomize` provisioner to see and interact with the bundle.
+
+Supported source types for a helm bundle currently include the following:
```

Comment:
nit: "Supported source types for a helm bundle ...".

Reply:
Thanks!